client/src/lib/routes.js

```diff
@@ -59,6 +64,15 @@ Status: 200 OK
 }
 */
 
+export const getInfo = (callback) => {
+  // console.log('fetching styles');
+  const path = `${apiPath}/products/${productID}`;
+  return fetch(path)
+    .then((result) => result.json())
```

Comment:
Not sure you have to parse the result coming back. I think this gets done on the server side... Could be wrong though lol not sure..

Reply:
`results.json()` is what I mean by this

Reply:
To extract the JSON body content from the response when using fetch, we use the json() method (defined on the Body, which is implemented by both the Request and Response objects.)